components/ActivityExplorerDialog.vue

```
@@ -0,0 +1,204 @@
+<script setup lang="ts">
+const isActivityDialogOpen = defineModel<boolean>("isActivityDialogOpen");
+const props = defineProps<{ board: DetailedBoard }>();
+
+const route = useRoute();
+const toast = useToast();
+
+const logs = ref<ActivityLog[]>([]);
+const logsCount = ref<number>(0);
+const firstElement = ref<number>(0);
+const isLoading = ref<boolean>(false);
+const userFilter = ref<any>();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)